src/test/resources/movie-tests.adoc

```diff
@@ -857,6 +859,107 @@ RETURN  movie  {  .title  } AS  movie
 ORDER BY movie.year DESC LIMIT $movieFirst
 ----
 
+=== Deeply nested orderBy
+
+.GraphQL-Query
+[source,graphql]
+----
+{
+  Movie(orderBy:title_desc) {
+    title
+    actors(orderBy:name_desc) {
+      name
+      movies(orderBy:[title_asc, title_desc]) {
```

Comment:
should that be allowed to sort by the same property twice in two ways?

Reply:
This test is taken over form the js-library. No it does not make any sense. I adjust it.